src/cogent3/maths/stats/special.py

```diff
@@ -179,8 +184,9 @@ def ln_binomial(successes, trials, prob):
     Note: no error checking (expects to be called through binomial_exact())
     """
     prob = fix_rounding_error(prob)
+    result_of_ln_combinations = lgam(trials + 1) - lgam(successes + 1) - lgam(trials - successes + 1)
```

Comment:
While this no longer calls the deprecated function, it should use `scipy.special.comb()` instead.

Reply:
I don't really think the `lgam()` function is deprecated, and the function is sophisticated enough that it can't simply be replaced by `scipy.special.comb()`.

Reply:
I take your point that `comb` is not a dropin replacement, however `lgam()` will eventually be deprecated. I suggest the following then:
```python
from scipy.special import loggamma # import statement inside the function, since we will be removing this function too

result_of_ln_combinations = loggamma(trials + 1) - loggamma(successes + 1) - loggamma(trials - successes + 1)
```